.circleci/config.yml

```diff
@@ -105,7 +105,7 @@ install_dep_pytorch_stable: &install_dep_pytorch_stable
         # check if we have restored venv cache (/home/circleci/venv) correctly, if so, just skip
         if [ -f /home/circleci/venv/check_version.py ]; then python /home/circleci/venv/check_version.py torch eq 1.11 && exit 0; fi
         # start installing
-        pip install --progress-bar off torch==1.12.0+cu113 torchvision==0.13.0+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
+        pip install --progress-bar off torch==1.12.1+cu113 torchvision==0.13.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
```

Comment:
we should test the latest stable version, which is 1.13 here. Overall, we test 3 versions:

lts: 1.8.x
stable: 1.13.x now
nightly: whatever next version

Reply:
For stable, I will do
`pip install --progress-bar off torch==1.13.0 torchvision==0.14.0`.
For nightly versions, I only see builds for cu116 or cu117. That's fine?